### DISCUSSION TITLE
What is the recommended way to create mathematical definitions?

### DISCUSSION THREAD

**Discussion by kedarmhaswade** [Digital-Media/HagenbergThesis#167] [General]
Should I proceed with the [usual](https://tex.stackexchange.com/a/385706/64425) `\usepackage{amsthm}` in my `main.tex` (because this package has a definition environment)? Is there another way that follows the Hagenberg _spirit_ of doing things?

**Comment by imagingbook** [comment]
`HagenbergThesis` has no special provision (nor "spirit") for math theorems. Just use whatever package you like and hope that it works as expected.
